homeassistant/components/jellyfin/__init__.py

```diff
@@ -0,0 +1,48 @@
+"""The Jellyfin integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.jellyfin.config_flow import (
+    CannotConnect,
+    InvalidAuth,
+    validate_input,
+)
+from homeassistant.components.jellyfin.const import DATA_CLIENT, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Jellyfin from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    try:
+        client = await validate_input(hass, dict(entry.data))
+    except CannotConnect as ex:
+        raise ConfigEntryNotReady("Cannot connect to Jellyfin server") from ex
+    except InvalidAuth as ex:
+        raise ConfigEntryAuthFailed("Failed to login to Jellyfin server") from ex
```

Comment:
This should return False. ConfigEntryAuthFailed starts the reauth flow, this should be added in another PR.
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#reauthentication

https://github.com/home-assistant/core/blob/51be2f860a253b97fae4d82897aea39b0b375c8c/homeassistant/config_entries.py#L276-L291

Reply:
Thanks for the information. I'll look at reauthentication later on. For now the setup will return false when the credentials are invalidated.